app/view_objects/daily_progress_digest.rb

```diff
@@ -5,19 +5,19 @@ def initialize options
   end
 
   def all_items_completed?
-    options[:remaining] == 0
+    options[:remaining_amount] == 0
   end
 
   def no_items_done?
-    options[:completed] == 0
+    options[:completed_amount] == 0
   end
```

Comment:
@vlasiak What about to reuse `completed_number` and `remaining_number` methods here? Also there is cool method to check for [zero](http://ruby-doc.org/core-2.2.0/Numeric.html#method-i-zero-3F) :) You may use it if you want.